component/Buttons/button.css

```diff
@@ -0,0 +1,60 @@
+.button{
+    background-color:var(--primary-color);
+    color:var(--white-color);
+    border-radius:3px;
+    border:0px solid var(--white-color);
+    font-size: 1.8rem;
+    font-weight: 400;
+    margin:2rem 2rem;
+    width:12vw;
+    height:6vh;
+}
+.button:hover{
+    cursor:pointer;
+    background-color: var(--primary-hover);
+}
+.outline-button{
+    color:var(--primary-color);
+    background-color: transparent;
+    border:1px solid var(--primary-color);
+}
+.outline-button:hover{
+    cursor:pointer;
+    background-color: var(--primary-hover);
+    color:var(--white-color)
+}
+.link-button{
+    background-color: var(--link);
+}
+.link-button:hover{
+    background-color: var(--link-hover);
+    cursor:pointer;
+}
+.text-link{
+    text-decoration: underline;
+    font-size: 2.5rem;
+    font-weight: 400;
+    color:var(--black);
+    border:0px solid var(--white-color);
```

Comment:
You can apply border as none, if you don't want the button border.

Reply:
ok done

Reply:
done